src/en/space-station-14/art.md

```diff
@@ -1,5 +1,233 @@
+Instead of depicting some kind of generic 'credit chit,' using paper bills is immediately recognizable.
+If you're trying to communicate the purpose of something, creating a visual association with something the player is likely familiar with can help them intuitively understand its function.
+* * If you're make a sci-fi generator, making it look similar to a real-life generator helps people understand it without close examination.
+* * Don't worry about the logistical side of retro technology.
+Any amount of ancient tech can be hand-waved away with budget restrictions and lazy scientists.
+Adhering to a personal concept of lore is secondary to serving the style.
+
+### Readability and Game Sprites
+
+* Keep the size of assets **proportional** to similar objects. 
+Obviously items will look over-sized compared to mobs, but making sure that the entire canvas isn't filled by a small object helps establish a relative scale.
+It can be confusing when an item that looks small when held appears significantly larger when dropped to the ground.
+
+
+* **Clothing** shouldn't cover excessive amounts of a mob or areas that shouldn't be covered by that region.
+Glasses shouldn't engulf an entire head and a hat shouldn't come down to the shoulders.
+Clothing should ideally have minimal overlap with other items so as to reduce visual clutter when many items are worn.
+
+
+* Mobs and direction-sensitive structures should contain **directional sprites** (directionals) for all 4 cardinal directions.
+When making directionals, the height of the asset should remain relatively constant and all details should be preserved for a consistent visual shape.
+Likewise, the color and placement of shadows should also shift to better depict the item turning in 3D space.
+
+## Technique
+
+### Perspective
+
+* **Structures** in the game world should be in a 3/4 perspective.
+This includes various machines, furniture, and decor.
+A 3/4's perspective means that they are angled straight forward with the viewer looking down on them from above. 
+This gives you a clear view of the top as well as the front of the object.
+
+
+* **Mobs**, **clothing**, and **items** are rendered in a flat perspective. 
+A flat perspective can be thought of as looking at the image straight-on at eye level.
+This maximizes detail while minimizing any perspective-based distortion.
+
+
+* **Floor tiles** and **ground** textures similarly lack any particular perspective.
+The actual floor itself should be a simple birds-eye perspective but any small detailing like stones or pits can employ a more conventional flat perspective.
+
+
+* **Walls** and wall-like structures (windows, grilles, etc.) are rendered in a exaggerated top-down perspective typically referred to as _classic_.
+Walls shouldn't have any kind of 3/4 perspective and, while the sides may be technically visible, the focus is on the "top" of the wall.
```

Comment:
This section may imply that we want people to make walls look like material walls / ore walls which have visible "innards," which is *DEFINITELY* not the case.
With that being said, walls are extremely hard to do and I'm not really sure how to convey how I think walls should probably usually look. Perhaps it may be best to emphasize that walls don't really have much of a perspective to begin with...? Either that, or just say "top-down" without elaborating.